### ISSUE TITLE
Clarification on Resistance and Persistent Damage

### ISSUE THREAD

**Issue by Hassurunous** [openlegend/core-rules#178]
"This damage automatically bypasses the afflicted character’s defenses but it can be reduced by any resistance to damage of a certain type (see the Resistance boon). "

It seems to imply that the damage can be reduced, but the Resistance boon doesn't say that it reduces damage, at least not until level 9. I think this might be an area where clarification would help. 

"Power Level 3 - Your defense scores are increased by 3 against the chosen attack type.

Power Level 5 - Your defense scores are increased by 6 against the chosen attack type.

Power Level 7 - Your defense scores are increased by 9 against the chosen attack type.

Power Level 9 - You are immune to damage and harmful effects from the chosen attack type."

If it is to be assumed that Resistance or Energy Resistance should grant some damage reduction from Persistent Damage, we should clarify that, possibly in both the Bane and the Boon(s).

**Comment by EliManthey** [comment]
Additionally it was brought up that multiple instances of a bane COULD stack as long as they are from different sources. I haven't been through all of the banes to see how broken this is but it is definitely a discussion topic.

**Comment by Hassurunous** [comment]
I think Bane stacking could be a separate issue, but I agree that it is something that should be discussed.

**Comment by brianfeister** [comment]
We can just add "unless you have damage immunity from the Resistance boon"

**Comment by Hassurunous** [comment]
We were talking more along the lines of how the Resistance should probably reduce the damage you take from Persistent Damage. Normally, because it increases defenses, this is accounted for. However, with Persistent Damage, you bypass defenses, meaning that the Resistance bonus is doing exactly 0 for you as long as the initial hit goes through. 

An example of why this is weird and clunky: 

I'm fighting a Banshee. It has Resistance level 3 to Entropy based on its undead nature, giving it a Toughness of 18 vs Entropy-based attacks. The party Necromancer uses a Bane attack to inflict Persistent Damage with Entropy. He rolls a 19, meaning that his attack hits. The Necromancer has an Entropy score of 6, which means that the Banshee is now taking 2d6 Entropy damage every turn until she resists. 

My natural instinct on this event would be that the Banshee would take 2d6-3 or something like that, to offset the Entropy damage, seeing as she has Resistance to it. 

Again, with anything but Persistent Damage, this point is moot, as the extra Defense equates to taking less damage, thanks to OL's dice system. With Persistent Damage however, Resistance becomes basically worthless unless its Power Level 9, at which point the creature is completely immune to it.

**Comment by brianfeister** [comment]
Gotcha. Good catch. Yeah, in case a future unknown bane comes along that does damage, we should just add 

> Your defense scores are increased by 3 against the chosen attack type and the first 3 damage of any non-attack roll sources is prevented.

**Comment by Hassurunous** [comment]
The description is apt, but, pretending for a moment to be a new player, casting Persistent Damage is a "Bane Attack", and it might be confusing that the damage that happens later, which is caused by that attack, is not, in fact, an "attack roll" but is instead a "non-attack roll" caused by an attack. 

Obviously, the rules say that this is not an "attack roll" per se, but for newer players (and newer GMs) this could be a potential point of confusion. 

Just a clarification fix. Perhaps something as simple as: 

> Reduce the damage done by any attack of the chosen type by 3. 

Or, to keep from using a "-X" or "+X" modifier system, we could simply add an example to the description to clarify, such as the Banshee example I listed above.

**Comment by brianfeister** [comment]
I'm open to suggestions, but it used to say 

> reduce damage of the chosen type by X

We had to change it because that leaves confusion about the threshold for triggering banes. If you don't specify the increase to defense scores, then it's unclear whether banes trigger on 7 or 10 damage.

**Comment by Elvenoob** [comment]
You could manage that by adding; ", which might prevent that attack from also adding a bane if the damage is reduced below 10."

**Comment by istabosz** [comment]
So, I actually think that we don't need to change anything here. Resistance is providing a benefit. It is increasing the target's defense and thus making it harder to apply the persistent damage. If we were to also reduce the damage per round, that would be making Resistance doubly effective.

A parallel example is "Demoralized". If I have resistance up vs. mental attacks, and someone attacks me with demoralized, my benefit from resistance comes from me having increased defense versus the bane attack. If the attack hits, I don't also get to reduce the penalty applied by demoralized.

**Comment by brianfeister** [comment]
Great point @istabosz. Agreed.
